Mathlib/Combinatorics/SimpleGraph/Metric.lean

```diff
@@ -183,10 +187,11 @@ protected theorem Connected.exists_walk_length_eq_dist (hconn : G.Connected) (u
 theorem dist_le (p : G.Walk u v) : G.dist u v ≤ p.length :=
   dist_eq_sInf ▸ Nat.sInf_le ⟨p, rfl⟩
 
-@[simp]
+@[simp, grind .]
```

Comment:
I'm a bit dubious about marking this for `grind` (I'm even a bit dubious about the `simp` tag). How about we leave this alone and just write `grind [dist_eq_zero_iff_eq_or_not_reachable]` below where necessary.
```suggestion
@[simp]
```